aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -385,10 +386,65 @@ torch::class_<LinearPackedParamsBase> register_linear_params() {
   return register_linear_params;
 }
 
+torch::class_<EmbeddingPackedParamsBase> register_embedding_params() {
+  // Type for __getstate__/__setstate__ serialization
+  //
+  // Element 0 is a string to say what kind of EmbeddingParam this is
```

Comment:
also I feel it might be better to have an int(enum) to indicate the type, since this will be serialized for each instance of embedding bag packed params